pkgs/tools/admin/lxd/default.nix

```diff
@@ -19,6 +19,11 @@ buildGoPackage rec {
     sha256 = "0sxkyjayn7yyiy9kvbdlpkl58lwsl2rhlxnncg628f2kad2zgkdx";
   };
 
+  patchPhase = ''
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
Could you give some additional information why this is the better option?

Reply:
usually it's better not to override the whole `patchPhase` because it may conflict with `patches`